plugin/storage/es/spanstore/reader.go

```diff
@@ -124,6 +123,7 @@ func NewSpanReader(p SpanReaderParams) *SpanReader {
 	if p.UseReadWriteAliases {
 		maxSpanAge = rolloverMaxSpanAge
 	}
+	archive := maxSpanAge == 0
```

Comment:
@yurishkuro do we want to remove the concept of archive altogether? should we call this variable something else?

Reply:
yes, in the reader / writer (only leave it in Factory for now)